locale/en/docs/guides/debugging-getting-started.md

```diff
@@ -77,38 +77,10 @@ info on these follows:
 * The latest version can also be installed independently (e.g. `npm install -g node-inspect`)
   and used with `node-inspect myscript.js`.
 
-#### [Chrome DevTools](https://github.com/ChromeDevTools/devtools-frontend) 55+
-
-* **Option 1**: Open `chrome://inspect` in a Chromium-based
-  browser. Click the Configure button and ensure your target host and port
-  are listed.
-* **Option 2**: Copy the `devtoolsFrontendUrl` from the output of `/json/list`
-  (see above) or the --inspect hint text and paste into Chrome.
-
-#### [Visual Studio Code](https://github.com/microsoft/vscode) 1.10+
```

Comment:
I agree it is not our place to recommend these tools so +1 on removing them.

Reply:
[VSCodium](https://github.com/VSCodium/vscodium) is VSCode without the trackers.

Reply:
Yes though why is it our place to recommend an IDE anyway?

Reply:
I don't know, but I thought the scope was about third-party privacy issues, not about _not_ recommending tools.